plugins/generator-1.18.1/forge-1.18.1/templates/tool.java.ftl

```diff
@@ -149,51 +148,13 @@ public class ${name}Item extends ${data.toolType?replace("Spade", "Shovel")?repl
     </#if>
 
 	<#if data.toolType=="MultiTool">

```

Comment:
It will look better if both conditional branches keep the same statements order:
```freemarker
	<#if data.toolType=="MultiTool">
		<@onEntityHitWith data.onEntityHitWith, true/>

		<@onBlockDestroyedWith data.onBlockDestroyedWithTool, true/>
	<#else>
		<@onBlockDestroyedWith data.onBlockDestroyedWithTool/>

		<@onEntityHitWith data.onEntityHitWith/>
    </#if>
```
<kbd>-></kbd>
```freemarker
	<#if data.toolType=="MultiTool">
		<@onBlockDestroyedWith data.onBlockDestroyedWithTool, true/>

		<@onEntityHitWith data.onEntityHitWith, true/>
	<#else>
		<@onBlockDestroyedWith data.onBlockDestroyedWithTool/>

		<@onEntityHitWith data.onEntityHitWith/>
	</#if>
```